### PR TITLE
feat: add zksync to deployments for testing purposes

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes introduced:**
- Adds and configures zkSync networks 
- Adds zkSync to aave markets with placeholder values for right now
- Updates HH config for zkSync to compile contracts and libraries
- Updates readme with relevant info for zkSync 

This PR comes first, and then the next PR to follow will include zkSync specific deployment scripts that will make use of our Deployer system contract.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +51,8 @@
     "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.3.0",
-    "typescript": "^4.3.2"
+    "typescript": "^4.3.2",
+    "zksync-web3": "^0.14.3"
```
Comment (MexicanAce) [inline package.json:55]: Should we be using `zksync-ethers`?

File: package.json [hunk 2]
```diff
@@ -25,6 +25,8 @@
     "@aave/core-v3": "^1.19.0",
     "@aave/periphery-v3": "^2.4.1",
     "@aave/safety-module": "^1.0.3",
+    "@matterlabs/hardhat-zksync-deploy": "^0.6.6",
+    "@matterlabs/hardhat-zksync-solc": "^1.0.4",
```
Comment (MexicanAce) [inline package.json:29]: Will `1.x.x` work with Ethers v5?

File: package.json (hunk 1 above)
Comment (dutterbutter) [reply package.json:55]: truly dont trust it just yet with plugin dependencies, I can update to test.

File: package.json (hunk 2 above)
Comment (dutterbutter) [reply package.json:29]: Don't know yet, thats next part. I am hopeful.

Comment (dutterbutter) [reply package.json:29]: I need 1.0.4 so might need to update. IDK yet.

File: package.json (hunk 1 above)
Comment (dutterbutter) [reply package.json:55]: let me do that in the next PR, as thats where i will actually make use of the package with deployment scripts.

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**